RELEASES.md

```diff
@@ -1,3 +1,71 @@
+Version 1.22.0 (2017-11-23)
+==========================
+
+Language
+--------
+- [`non_snake_case` lint now allows extern no-mangle functions][44966]
+- [Now accepts underscores in unicode escapes][43716]
+- [`#![feature(const_fn)]` is now no longer required for
+  calling const functions.][43017] It's still required for creating
+  constant functions.
```

Comment:
Individual const functions are still unstable one by one — is there any of them that are stable in Rust 1.22?

Reply:
I think `mem::size_of`?

Reply:
That would be amazing, but size_of is not stabilized in current nighly.

Reply:
cc @durka Do you know any?

Reply:
There are none so far. We only put the infrastructure in place for doing them individually. I thought size_of was imminent though, what happened?

Reply:
@Aaronepower Can we remove this line about const functions? Since it has no user-visible effect in Rust 1.22

Reply:
@bluss Well shouldn't it still be included since this is when it was stabilised?

Reply:
What's misleading about the current bullet point is you still need a feature gate for calling const fns, just a different one (in fact a different one per fn now). So you can't do anything on stable that you couldn't before.

Reply:
I agree with durka here, it's nothing user-visible (unfortunately) for the Rust 1.22.0 release.